### PR TITLE
Minor improvements to docs

### PR DESCRIPTION
**Pull request by weitzhandler**
## PR Type

- Minor documentation proofreading

<hr/>

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: doc/articles/debugging-wasm.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 Debugging WebAssembly via Google Chrome is experimentally supported by the Uno Platform. We HIGHLY recommend that you use [Google Chrome Canary](https://www.google.com/chrome/canary/).  Step-through debugging (in, out, over), breakpoints, inspection of run-time locals and viewing .NET source code from the developer tools works. Additional capabilities and browser support will become available as Microsoft adds [support for them to mono](https://github.com/mono/mono/tree/master/sdks/wasm).
 
--  Make your project the startup project (right-click **set as startup** in Solution Explorer)
+- Make your WASM project the startup project (right-click **set as startup** in Solution Explorer)
```
Comment (jeromelaban) [inline doc/articles/debugging-wasm.md]:
```suggestion
- Make your Wasm project the startup project (right-click **set as startup** in Solution Explorer)
```

File: doc/articles/debugging-wasm.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ Debugging WebAssembly via Google Chrome is experimentally supported by the Uno P
    - Make sure script debugging is disabled<br/>
    ![IIS express settings](Assets/quick-start/wasm-debugging-iis-express.png)
 
-- Start the debugging session using `Ctrl+F5` (`F5` will work, but the debugging experience won't be in Visual Studio)
-- Once your application has started, press `Alt+Shift+D` (in Chrome, on your application's tab)
+- Start the debugging session using <kbd>Ctrl</kbd><kbd>F5</kbd> or _Debug_ > _Start Without Debugging_ from the menu, (<kbd>F5</kbd> will work, but the debugging experience won't be in Visual Studio)
```
Comment (jeromelaban) [inline doc/articles/debugging-wasm.md]: Nice ;)

File: doc/articles/debugging-wasm.md [hunk 3]
```diff
@@ -36,10 +36,10 @@ Debugging WebAssembly via Google Chrome is experimentally supported by the Uno P
 > You may simply install _Chrome Beta_ or _Chrome Canary_ and use them instead.
 > * Sometimes, you may have a problem removing a breakpoint from code (it's crashing the debugger).
 > You can remove them in the _Breakpoints list_ instead.
-> * Once _IIS Express_ is launched, no need to press `Ctrl+F5` again: you simply need to rebuild your
-> _Wasm_ head and refresh it in the browser.
-> * **To refresh an app**, you should use the debugger tab and press the _refresh button_ in the content.
+> * Once _IIS Express_ is launched, no need to press <kbd>Ctrl</kbd><kbd>F5</kbd> again: you simply need to rebuild your
+> _WASM_ head and refresh it in the browser.
```
Comment (jeromelaban) [inline doc/articles/debugging-wasm.md]:
```suggestion
> _Wasm_ head and refresh it in the browser.
```

Comment (weitzhandler) [reply doc/articles/debugging-wasm.md]: I think when used in a documentation context (e.g. not in code), it's spelled WASM, as acronyms, no?

File: doc/articles/debugging-wasm.md (hunk 1 above)
Comment (weitzhandler) [reply doc/articles/debugging-wasm.md]: See [my comment](https://github.com/unoplatform/uno/pull/1811#discussion_r333949980).

File: doc/articles/debugging-wasm.md (hunk 3 above)
Comment (weitzhandler) [reply doc/articles/debugging-wasm.md]: Like [LINQ](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/). It's spelled _LINQ_, but coded as `Linq`.

**Pull request merged**